go.mod

```diff
@@ -32,7 +32,6 @@ require (
 	github.com/smartystreets/assertions v1.1.1 // indirect
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/viper v1.8.1
-	go.mongodb.org/mongo-driver v1.1.2 // indirect
```

Comment:
that's an indirect dep - it's a bit clunky style of introduction not sure why `go mod` adds it. 

Although your intentions are right in the PR - the problem is it might still get again added since it's some kind of dependency management behavior. 

Let's take in and see.

Reply:
go mod why said it’s coming from spf13/viper v1.7. We upgraded to v1.8.1 in a previous PR, but go mod tidy didn’t remove this line. 

This time, go mod why says, this dependency is not needed. Even without removing the line. That’s why we are ok with manually removing it